packages/compiler-sfc/src/template/transformAssetUrl.ts

```diff
@@ -42,7 +42,13 @@ export const defaultAssetUrlOptions: Required<AssetURLOptions> = {
     source: ['src'],
     img: ['src'],
     image: ['xlink:href', 'href'],
-    use: ['xlink:href', 'href']
+    use: ['xlink:href', 'href'],
+    audio: ['src'],
+    object: ['data'],
+    embed: ['src'],
+    track: ['src'],
+    link: ['href'],
+    iframe: ['src']
```

Comment:
While it is possible to have base64 as the `iframe` `src`, this use case is so niche that I doubt adding it here would be helpful.
On the other hand, processing `iframe` `src` as an imported asset module might cause some confusing bugs for users. So I suggest dropping this line.